crypto/bio/bss_file.c

```diff
@@ -319,8 +319,8 @@ static long file_ctrl(BIO *b, int cmd, long num, void *ptr)
     case BIO_CTRL_FLUSH:
         if (b->flags & BIO_FLAGS_UPLINK)
             UP_fflush(b->ptr);
```

Comment:
If you are going to check the fflush return information it should also be done here in the uplink case too